src/Main.elm

```diff
@@ -46,6 +51,17 @@ init =
         watchDate movie =
             Movie.watchDate movie
                 |> Maybe.withDefault (date 1800 1 1)
+
+        genres =
+            MovieList.movies
+                |> List.map .genres
+                |> List.map (Set.map String.toLower)
+                |> List.foldr Set.union Set.empty
+
+        selectOptions set =
+            set
+                |> Set.map (\m -> ( m, toUpper m ))
```

Comment:
Takes each item in a `Set<string>` and turns it into a tuple of `("thestring", "Thestring")`